nextcloudappstore/settings/base.py

```diff
@@ -198,3 +198,5 @@
 
 LOGIN_REDIRECT_URL = 'home'
 LOGIN_URL = 'account_login'
+
+PLATFORM_VERSIONS = []
```

Comment:
Right, so the basic issue is that ownCloud has a weird versioning schema, as in: they don't follow semver or any other commonly used version schema.

As a user you want to see download buttons for your base version, e.g. Releases compatible with 9.0, Relases compatible with 9.1, etc

This is the thing that needs to be rendered on the page.